sources/platform/actors/development/quick_start/start_web_ide.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Web IDE
+sidebar_position: 2
+description: Create you first Actor using the web IDE in Apify Console.
+slug: /actors/development/quick-start/web-ide
+---
+
+# Development in web IDE
+
+**Create you first Actor using the web IDE in Apify Console.**
+
+---
+
+## 1. Create the Actor
+
+> You will need an Apify account to complete this tutorial. If you don't have one, you can [sign-up for a free account](https://console.apify.com/sign-up) on the Apify website.
+
+After you sign in to [Apify Console](https://console.apify.com), navigate to the [Actors](https://console.apify.com/actors) section on your left panel. Then, click the **Create new** button at the top right corner of the page.
+
+![Create Actor](./images/actor-create-button.png)
+
+You will be redirected to a page containing various Actor development templates for popular languages such as JavaScript, TypeScript, and Python. You can choose the template that best suits your technology stack. For demonstration purposes, let's choose "PuppeteerCrawler in Javascript".
```

Comment:
Same here, PuppeteerCrawler in JS is no longer there and the screenshots are outdated.

Reply:
I am moving this into the following round of changes, as this text was already there and was just move to a different location.